Dockerfile

```diff
@@ -426,7 +426,7 @@ RUN \
   wget -O vid.stab.tar.gz "$VIDSTAB_URL" && \
   echo "$VIDSTAB_SHA256  vid.stab.tar.gz" | sha256sum --status -c - && \
   tar xf vid.stab.tar.gz && \
-  cd vid.stab-* && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DUSE_OMP=ON . && \
+  cd vid.stab-* && sed -i 's/include (FindSSE)/if(CMAKE_SYSTEM_ARCH MATCHES "amd64")\ninclude (FindSSE)\nendif()/' ./CMakeLists.txt && cmake -DCMAKE_SYSTEM_ARCH=$(arch) -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DUSE_OMP=ON . && \
```

Comment:
I suspect this only happens when building inside an emulated (arm) container when /proc is from the host (amd64 with sse support), if the host was arm also it would probably work. I've reported this https://github.com/docker/for-mac/issues/5245 but seems complicated to fix so we probably have to workaround it somehow.

Luckely it seem CMAKE_SYSTEM_ARCH work fine in emulated container so your suggestion i think make sense.

Reply:
Maybe it's better to reverse condition, to exclude this step only on arm64 arch. WDYT?

Reply:
Hmm not sure, i think SSE is an intel only thing?

Reply:
It seems like it this. 

But it's a little strange that we check for Intel feature on architecture containing "AMD" in its name. :)

Reply:
Yeah the name is a bit confusing, it was amd that invented the x86-64 architecture that even intel uses now :)

Reply:
Same here document why it's done